src/tests/ftest/pool/InfoTests.py

```diff
@@ -25,12 +25,9 @@
 import sys
 import json
 import logging
-from avocado import Test, main
+from apricot import Test
+from avocado import main
```

Comment:
(pylint-import-error) Unable to import 'avocado'

(pylint-unused-import) Unused main imported from avocado